run-dljc.sh

```diff
@@ -23,40 +25,32 @@
 #
 # -i infile : read the list of repositories to use from the file $infile. Each
 #             line should contain the (https) url of the git repository on
-#             GitHub and the commit hash to use, separated by whitespace. If the
-#             repository's owner is you (see -u flag), then each line owned by
-#             you must be followed by the original github repository.
-#
-# -c checkers : a comma-separated list of typecheckers to run
+#             GitHub and the commit hash to use, separated by whitespace.
+#             Repositories must be of the form:
+#             https://github.com/username/repository - the script is reliant
+#             on the number of slashes, so excluding https:// is an error.
+#             
+#             If the repository's owner is the user specified by the -u flag,
+#             then each line owned by that user must contain a third element,
+#             the original github repository.
 #
 # The meaning of each optional argument is:
 #
-# -l lib : a colon-separated list of jar files which should be added to the
-#          java classpath when doing typechecking. Use this for the dependencies
-#          of any custom typecheckers.
-#
-# -q quals : a colon-separated list of the jar files containing annotations used
-#            by custom checkers
-#
-# -s stubs : a colon-separated list of stub files
-#
 # -u user : the GitHub user to consider the "owner" for repositories that have
 #           been forked and modified. These repositories must have a third entry
 #           in the infile indicating their origin. Default is "kelloggm".
```

Comment:
I think you can avoid defaulting to "kelloggm".  Just use `$USER`.

Reply:
I had forgotten about this default. I chose that default for my own convenience :)

I've made the change.